incubator/model-avro/src/main/java/io/aklivity/zilla/runtime/model/avro/internal/AvroModelHandler.java

```diff
@@ -81,7 +81,7 @@ protected AvroModelHandler(
         this.encoder = encoderFactory.binaryEncoder(EMPTY_OUTPUT_STREAM, null);
         CatalogedConfig cataloged = config.cataloged.get(0);
         this.handler = supplyCatalog.apply(cataloged.id);
-        this.catalog = cataloged.schemas.size() != 0 ? cataloged.schemas.get(0) : null;
+        this.catalog = cataloged.schemas.size() != 0 ? cataloged.schemas.stream().findFirst().get() : null;
```

Comment:
We can revert this if reverting to List, right?

Reply:
Hmm, I have resolved this but not sure why it is not updated locally I don't have this change.

Reply:
Figured it out I was debugging issue reported by barnabas and accidentally made changes into that branch